hail/python/hailtop/hailctl/dev/benchmark/run/shuffle_benchmarks.py

```diff
@@ -0,0 +1,38 @@
+from os import path
+from tempfile import TemporaryDirectory, NamedTemporaryFile
+
+import hail as hl
+from .utils import benchmark, resource
+
+
+@benchmark
+def shuffle__key_rows_by_mt():
+    mt = hl.read_matrix_table(resource('profile.mt'))
+    mt = mt.annotate_rows(reversed_position_locus=hl.struct(
+        contig=mt.locus.contig,
+        position=-mt.locus.position))
+    mt = mt.key_rows_by(mt.reversed_position_locus)
+    mt._force_count_rows()
+
+
+@benchmark
+def shuffle__order_by_10m_int():
+    t = hl.utils.range_table(10_000_000, n_partitions=100)
+    t = t.order_by(-t.idx)
+    t._force_count()
+
+
+@benchmark
+def shuffle__key_rows_by_4096_byte_rows():
+    mt = hl.utils.range_matrix_table(100_000, (1 << 12) // 4)
```

Comment:
might want to set a n_partitions here to something like 16, since it'll use the default of 1 when running on one core

Reply:
and they'll be a bit bigger than 4k with missing bits, is that OK?

Reply:
yeah, I'm more getting at the rough size of the data being transferred.